src/features/amUI/userRightsPage/DelegationModal/ResourceInfo.tsx

```diff
@@ -49,22 +49,50 @@ export const ResourceInfo = ({ resource, toParty, onDelegate }: ResourceInfoProp
   const [delegateRights] = useDelegateRightsMutation();
   const [rights, setRights] = useState<ChipRight[]>([]);
   const { openSnackbar } = useSnackbar();
+  const { data: reportee } = useGetReporteeQuery();
+
+  const [missingAccessMessage, setMissingAccessMessage] = useState<string | null>(null);
   const displayResourceAlert =
     error.isError ||
     resource?.delegable === false ||
     (rights.length > 0 && !rights.some((r) => r.delegable === true));
+
   const resourceRef: ResourceReference | null =
     resource !== undefined
       ? {
           resource: resource.authorizationReference,
         }
       : null;
 
+  const getMissingAccessMessage = (response: DelegationAccessResult[]) => {
+    const hasMissingRoleAccess = response.some((result) =>
+      result.details?.some((detail) => detail.code === 'MissingRoleAccess'),
+    );
+
+    const hasMissingSrrRightAccess = response.some(
+      (result) =>
+        !hasMissingRoleAccess &&
+        result.details?.some((detail) => detail.code === 'MissingSrrRightAccess'),
```

Comment:
Also, consider using the `ErrorCode` enum so we are less vulnerable to things like typos in string values

Reply:
👍